backends/events/sdl/sdl-events.cpp

```diff
@@ -900,8 +938,10 @@ bool SdlEventSource::remapKey(SDL_Event &ev, Common::Event &event) {
 void SdlEventSource::resetKeyboardEmulation(int16 x_max, int16 y_max) {
 	_km.x_max = x_max;
 	_km.y_max = y_max;
-	_km.delay_time = 25;
+	_km.delay_time = 12;
 	_km.last_time = 0;
+	_km.multiplier = 16;
```

Comment:
multiplier: this should not be changed you are right. I will make a commit to define it as const. It is just used to get more fine-grained control over velocity, so that the mouse pointer velocity can be less than 1 pixel per frame without becoming exactly equal to zero.

modifier: I want to keep it open for future use. For example, in the forthcoming Vita port, the modifier is changed by holding down the R-Trigger on the gamepad. Any other gamepad-centric platform code could potentially make use of this modifier (e.g. the R-Trigger, so that the mouse is slowed when holding R+analog stick).